README.md

```diff
@@ -1,40 +1,43 @@
 # PodCodar App
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+Este projeto tem por objetivo facilitar o processo de recepção de novos membros, por estarmos enfrentando uma grande dificuldade em recepcionar com qualidade a todos que estavam chegando, resolvemos então adicionar algumas funcionalidades que podem nos ajudar.
 
-## Getting Started
+## Instalação
 
-First, run the development server:
+1. Clone o repositório
+  - para fazer isso basta abrir o terminal e rodar o seguinte comando ```git clone https://github.com/podcodar/app.git```
+2. Execute o comando ```yarn```para instalar as dependências do projeto.
+  - lembre-se de navegar até a pasta onde o projeto se encontra e então quando la estiver, execute o comando acima! Ps: para navegar entre pastas pelo terminal basta usar o comando``` cd nome da pasta``` para entrar na pasta e ```cd ..``` para regredir.
 
+
+## Utilização
+  - Certifique-se de que os passos de instalação já estejam concluídos! E então execute um desses comandos,
 ```bash
 npm run dev
 # or
 yarn dev
-# or
-pnpm dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
-
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+Com o passo anterior feito, abra o link [http://localhost:3000](http://localhost:3000) para ver o resultado.
+
+## Funcionalidades
+
+- 
+- 
+- 
+- 
+
```

Comment:
This section will be too volatile. I suggest we remove it, so avoid changing this file constantly.